test/inspector/test-inspector-port-cluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const assert = require('assert');
+const cluster = require('cluster');
+const common = require('../common');
+const debuggerPort = common.PORT;
+const childProcess = require('child_process');
+
+common.skipIfInspectorDisabled();
+
+let offset = 0;
+
+/*
+ * This test suite checks that inspector port in cluster is incremented
+ * for different execArgv combinations
+ */
+
+function testRunnerMain() {
+  spawnMaster({
+    execArgv: ['--inspect'],
+    workers: [{expectedPort: 9230}]
+  });
+
+  let port = debuggerPort + offset++ * 10;
+
+  spawnMaster({
+    execArgv: [`--inspect=${port}`],
+    workers: [
```

Comment:
This is what I suggested doing https://github.com/nodejs/node/pull/13373/files#diff-4b2a51d5cbf4c8edb50bfd0f7016bb84R16 And just close the workers immediately.
In a way it's actually a more specific test to your code changes (manipulating argv, rather than making sure the worker starts)